docker-compose.yml

```
@@ -44,16 +43,15 @@ services:
-      MYSQL_PASSWORD: ${MYSQL_PASSWORD}
+      MYSQL_USER: merico
+      MYSQL_PASSWORD: merico
       TZ: UTC
-      ENCRYPTION_SECRET: ${ENCRYPTION_SECRET}
     restart: always
     depends_on:
       - mysql
 
   devlake:
-    image: devlake.docker.scarf.sh/apache/devlake:v1.0.1   
+    image: devlake.docker.scarf.sh/apache/devlake:v1.0.0   
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)